Source/OxyPlot/Rendering/OxyCircle.cs

```diff
@@ -0,0 +1,156 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="OxyCircle.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Describes the width, height, and point origin of a rectangle.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot
+{
+    using System;
+    using System.Globalization;
+    using System.Text;
+
+    /// <summary>
+    /// Describes the center and the radius of a circle.
+    /// </summary>
+    public struct OxyCircle : IOxyRegion, IEquatable<OxyCircle>, IFormattable
+    {
+
+        /// <summary>
+        /// Gets the center point of the circle.
+        /// </summary>
+        /// <value>The center.</value>
+        public ScreenPoint Center { get; private set; }
+
+        /// <summary>
+        /// Gets the radius of the circle
+        /// </summary>
+        public double Radius { get; private set; }
```

Comment:
These should be getter only (i.e. `{ get; }`) (only assigned in the constructor, so don't need a private setter)

Reply:
as above